src/lib/formats/cassimg.h

```diff
@@ -127,6 +127,7 @@ class cassette_image
 		uint32_t sample_frequency = 0;
 		int header_samples = 0;
 		int trailer_samples = 0;
+		int (*fill_wave_ext)(int16_t *, int, const uint8_t *, int) = nullptr;
```

Comment:
Alternatively I can change signature for all existing `fill_wave`

Reply:
I'm waiting for consensus. Have this ^^^ implemented locally or can easily refactor to loading error without crash if you insist.